policyengine_canada/variables/gov/provinces/qc/tax/income/credits/medical_expenses/qc_medical_expense_credit.py

```diff
@@ -0,0 +1,12 @@
+from policyengine_canada.model_api import *
+
+
+class qc_medical_expense_credit(Variable):
+    value_type = float
+    entity = Person
+    label = "Quebec medical expenses tax credit"
+    definition_period = YEAR
+    defined_for = "qc_medical_expense_credit_eligible"
+
+    def formula(person, period, parameters):
+        return person("medical_expenses", period)
```

Comment:
do filers get their full medical expenses deduction if they meet the requirements? Could you add a reference to this variable where it states that?

Reply:
I've re-edited this program. To be specific, I separated the non-refundable medical expense credit and calculate the refundable credit based on the non-refundable one. The previous one was based on the [table](https://www.revenuquebec.ca/en/citizens/income-tax-return/completing-your-income-tax-return/how-to-complete-your-income-tax-return/line-by-line-help/451-to-480-refund-or-balance-due/line-462/point-1/) on the website but I found maybe [formulas](https://www.revenuquebec.ca/documents/en/formulaires/tp/2022-12/TP-1.D.B-V%282022-12%29.pdf#page=2) are easier to understand.